clickstream/docs/index.rst

```diff
@@ -63,7 +72,17 @@ are quite large and depending on your network connection may take
         ksqldb-server     bash -c # Manually install ...   Up      0.0.0.0:8083->8083/tcp, 0.0.0.0:8088->8088/tcp
         schema-registry   /etc/confluent/docker/run        Up      8081/tcp                                      
         tools             /bin/bash                        Up                                                    
-        zookeeper         /etc/confluent/docker/run        Up      2181/tcp, 2888/tcp, 3888/tcp    
+        zookeeper         /etc/confluent/docker/run        Up      2181/tcp, 2888/tcp, 3888/tcp   
+
+
+#. Once you've confirmed everything is running in previous step, run the source connectors via ksqldb-cli 
+
+   .. code:: bash
+
+        docker-compose exec ksqldb-cli bash -c "ksql http://ksqldb-server:8088 <<EOF
+        run script '/scripts/create-connectors.sql';
+        exit ;
+        EOF"
 
 ---------------------------
 Browse the Clickstream Data
```

Comment:
`Browse the Clickstream Data` --> `Create the Clickstream Data`

Reply:
ack